client/components/theme/index.jsx

```diff
@@ -594,7 +594,7 @@ export class Theme extends Component {
 
 					<div
 						className={ classNames( 'theme__info', {
-							'has-pricing': !! upsellUrl,
```

Comment:
Assume this was dead code.  I searched for `has-pricing` in calypso and nothing came up, e.g.  that class doesn't exist in a stylesheet.

Reply:
I found that the class `has-pricing` is being replaced by `has-style-variations` in this same PR, which is the only place being used because originally it was added by us in [Theme Showcase: Enable theme card's more button](https://github.com/Automattic/wp-calypso/pull/71781).

Reply:
Ah, yes!  I missed that in `theme-showcase.scss`.  Thank you.